src/main/kotlin/com/viartemev/thewhiterabbit/channel/Channels.kt

```diff
@@ -67,6 +86,18 @@ suspend fun Connection.confirmChannel(block: suspend ConfirmChannel.() -> Unit):
         .also { block(it) }
 }
 
+suspend fun Connection.txChannel(block: suspend TxChannel.() -> Unit): TxChannel {
```

Comment:
We duplicate this code 3 times. Maybe we can add generic function?

Reply:
Will try to. Also I am in the process of adding more idiomatic using for transactions like `channel { transaction { publish ... }}`

Reply:
Sounds greate!